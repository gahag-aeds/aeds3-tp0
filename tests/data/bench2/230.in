230 115
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 
Sub 124 199
Min 39 114
Add 93 168
Max 92 167
Add 119 194
Min 145 220
Add 135 210
Max 18 93
Add 79 154
Min 62 137
Add 85 160
Min 45 120
Add 153 228
Max 54 129
Sub 6 81
Min 113 188
Sub 20 95
Sum 14 89
Add 87 162
Sum 49 124
Add 33 108
Sum 103 178
Sub 4 79
Min 33 108
Add 101 176
Max 118 193
Add 3 78
Sum 68 143
Add 143 218
Min 51 126
Sub 50 125
Max 113 188
Add 89 164
Sum 140 215
Sub 48 123
Sum 54 129
Sub 128 203
Sum 28 103
Add 69 144
Max 50 125
Add 87 162
Max 151 226
Sub 94 169
Sum 85 160
Sub 50 125
Min 99 174
Add 41 116
Max 69 144
Sub 112 187
Min 125 200
Sub 116 191
Sum 142 217
Sub 74 149
Min 117 192
Add 19 94
Max 20 95
Sub 78 153
Min 21 96
Sub 88 163
Min 67 142
Sub 70 145
Min 135 210
Add 25 100
Sum 4 79
Sub 120 195
Sum 70 145
Add 57 132
Min 91 166
Add 97 172
Min 123 198
Sub 140 215
Sum 30 105
Add 119 194
Sum 79 154
Sub 112 187
Max 15 90
Add 21 96
Min 153 228
Add 81 156
Max 131 206
Add 123 198
Sum 42 117
Sub 118 193
Max 40 115
Sub 2 77
Max 136 211
Sub 60 135
Min 80 155
Add 1 76
Sum 145 220
Sub 146 221
Sum 70 145
Add 125 200
Sum 14 89
Add 71 146
Sum 88 163
Add 83 158
Sum 126 201
Sub 24 99
Min 23 98
Sub 94 169
Min 9 84
Sub 50 125
Max 59 134
Add 85 160
Min 8 83
Add 71 146
Min 105 180
Sub 4 79
Max 149 224
Add 81 156
Sum 124 199
Add 37 112
Min 43 118
Sub 10 85
