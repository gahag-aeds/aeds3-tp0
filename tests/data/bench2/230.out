0
98
0
99
1
1
100
2
3509
3774
3916
-1
103
3894
0
104
4298
3657
3735
99
103
4042
8
99
1
4179
0
98
-1
1
0
3681
3514
4
-1
3901
3771
98
-2
99
3830
99
99
3
4036
3633
3686
3722
4086
0
0
97
-1
-2
98
4066
-1
